thoth/user_api/api_v1.py

```diff
@@ -189,7 +189,7 @@ def post_analyze(
     )
     analysis_by_digest_store = AnalysisByDigest()
     analysis_by_digest_store.connect()
-    analysis_by_digest_store.store_document(metadata["digest"], response)
```

Comment:
seems like the order is correct based on this
https://github.com/thoth-station/storages/blob/07600bf68209b8c09d84d12e6fe568a75b6c260e/thoth/storages/analyses_by_digest.py#L29

Reply:
Yeah it currently is (hence the `/hold`) The mypy fixes on storages I've been doing on storages uncovered that the underlying API is broken (see https://github.com/thoth-station/storages/issues/2689) TL;DR: `str` treated as `dict`. So the mentioned commit fixes that, and get rids of the method overrides which did not add much.

I've just done that PR at the same time because it's the only call sites I found searching accross the org on Github, and it was simpler to do it while it was fresh in my mind.